README.md

```diff
@@ -325,3 +325,14 @@ A better approach would be to work with a separate compiler phase, similar to wh
 
 If there's a compilation issue, the temporary script file will not be deleted and the error output will tell you it's path, in order to help with debugging.
 
+## Parameters cheat sheet: the most important ones
+Here's only the most important ones - run `scala-repl-pp --help` for all parameters.
+
+| parameter     | short         | description                           
+| ------------- | ------------- | --------------------------------------
+| `--predef`    | `-p`          | Import additional files
+| `--dep`       | `-d`          | Add dependencies via maven coordinates
+| `--repo`      | `-r`          | Add repositories to resolve dependencies
+| `--script`    |               | Execute given script
+| `--param`     | `-p`          | key/value pair for main function in script
```

Comment:
oh, just noticed: there's two `-p` shortcuts here...

Reply:
whoopsie, thanks for paying attention!